protostar/commands/deploy_account_command.py

```diff
@@ -45,7 +46,7 @@ def from_args(cls, args: Namespace):
 
         assert signer is not None
         return cls(
-            account_address=args.account_address,
+            account_address=AccountAddress(int(args.account_address)),
```

Comment:
why do we coerce it to int first? Shouldn't  be "from_user_input" used here?

Reply:
Argument parser converts this argument to `Address` first. I could avoid casting by introducing to `account_address`  type in `protostar_arg_type.py`